lib/rules/no-debug.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @fileoverview Don't allow debug() to be committed to the repository. 
+ * @author Ben Monro
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Don't allow `t.debug()` to be committed to the repository. ",
+            category: "Mistake Prevention",
+            recommended: true
+        },
+        fixable: null,
+        schema: []
+    },
+
+    create: function (context) {
+        return {
+            "CallExpression[callee.property.name='debug']"(node) {
```

Comment:
Can we check that the `.debug` call performed into the `test` function?

Reply:
I thought about that but I don't want them in roles out custom functions or page models either.  It's safer this way and if on the off chance it's reporting incorrectly you can use eslint disable